### ISSUE TITLE
Can't get voice commands to work on HomeassistantOS

### ISSUE THREAD

**Issue by Yersi88** [stanford-oval/almond-hassio-repository#15]
Hello, 

I installed Amond edge version on my HA raspberry but I can't get voice commans to work. 

I have audio configured like this using an USB microphone: 
![image](https://user-images.githubusercontent.com/77145281/137765558-342f3fc1-f7cf-4e78-9879-023bf7cbfddf.png)

When I try to use the wake word and talk to it nothing happens, there's no response and also nothing in the logs. 

What can I do the get the voice recognition to work?

**Comment by gcampax** [comment]
Can you post the full logs here? It is possible that audio is not initializing correctly.

**Comment by Yersi88** [comment]
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2021/10/18 16:07:05 [notice] 206#206: using the "epoll" event method
2021/10/18 16:07:05 [notice] 206#206: nginx/1.14.2
2021/10/18 16:07:05 [notice] 206#206: OS: Linux 5.10.17-v8
2021/10/18 16:07:05 [notice] 206#206: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2021/10/18 16:07:05 [notice] 206#206: start worker processes
2021/10/18 16:07:05 [notice] 206#206: start worker process 233
Frontend initialized in development mode
Failed to load translations: The expression evaluated to a falsy value:
  assert_1.default(fs.existsSync(modir))
Snowboy initialized
Failed to cache event sound: Operation not supported
Added device org.thingpedia.builtin.thingengine.server
Added device thingengine-own-server:bb7f0c3150bdb4de
Added device org.thingpedia.builtin.test
Added device org.thingpedia.builtin.thingengine.builtin
Activity monitor started
Activity monitor started idle timer 599999 ms
Engine started
Initializing dialogue policy for devices: 
Record stream is now creating
Record stream is now ready
```

This is what I was able to get from the log, if you give me some instructions I can get _fuller_ logs :)

**Comment by gcampax** [comment]
That's about as full as it gets. It looks like the audio input and the wakeword are initialized correctly. Do you see "Wakeword detected" in the log when you say the wakeword? The wakeword is "Computer" btw.

**Comment by Yersi88** [comment]
Nope, nothing in the log when I say the wakeword :(

**Comment by gcampax** [comment]
That means the wakeword is not being detected at all. One problem might be the wakeword model we're currently using (based on snowboy). Another problem might be the microphone volume being too low. There is currently no UI to change volumes in Home Assistant OS - this is a feature that would need to be added in the OS.

**Comment by Yersi88** [comment]
I first thought my microphone is bad, it's one of those conference types https://www.amazon.co.uk/gp/product/B08YYPH49R?psc=1 

But I tried it on my windows box and that worked no problem, very clear sound. 

On HA again I tried to talk loud and directly into it, still didn't detect the wake word. 


There is still a possiblity there might be some problem with the mic. I would try another one. Is there something that's known to work that I could buy?
